server/middlewares/errorHandler.js

```diff
@@ -0,0 +1,32 @@
+const {
+    NotFoundError,
+    UnauthorizedError,
+    AuthenticationError,
+    UnavailableServiceError,
+    BadRequestError } = require('../errorTypes/errorTypes')
+
+module.exports = handleError = (err, req, res, next) => {
+    switch (err.constructor) {
```

Comment:
no this is not right.
please read the task description carefully.
_implement middleware to extend res with method handleError(error)_
you need to extend res object with method handleError(error)

Reply:
Okay, I didn't understand how that should work, but I get it now and changed these next command to handleError